internal/kafka/test/integration/kafkas_test.go

```diff
@@ -804,15 +797,15 @@ func TestKafkaCreate_TooManyKafkas(t *testing.T) {
 		resp.Body.Close()
 	}
 
-	g.Expect(err).ToNot(gomega.HaveOccurred(), "Ung.Expected error occurred when creating kafka:  %v", err)
+	g.Expect(err).ToNot(gomega.HaveOccurred(), "Expected error occurred when creating kafka:  %v", err)
 
 	k.Name = mockKafkaName
 
 	_, resp, err = client.DefaultApi.CreateKafka(ctx, true, k)
 	if resp != nil {
 		resp.Body.Close()
 	}
-	g.Expect(err).To(gomega.HaveOccurred(), "g.Expecting error to be thrown when creating more kafkas than allowed by the cluster limit:  %v", err)
```

Comment:
There are other occurrences of `g.Expecting` in the file. Can we fix them?

Reply:
Good catch. Thanks, I've update several other occurrences. Can you have another look?